test/IDE/complete_opaque_result.swift

```diff
@@ -200,3 +203,36 @@ func postfixExpr() {
 // POSTFIX_TestProtocol_NODOT-DAG: BuiltinOperator/None:                = {#TestProtocol#}[#Void#]; name={{.*$}}
 // POSTFIX_TestProtocol_NODOT-DAG: Keyword[self]/CurrNominal:          .self[#TestProtocol#]; name={{.*$}}
 // POSTFIX_TestProtocol_NODOT-DAG: End completions
+
+protocol TestProtocol2 {
+  associatedtype Assoc: Comparable
+  func foo() -> Assoc
+  func bar() -> Assoc
+  func baz(x: @autoclosure () -> Assoc) -> (Assoc) -> Assoc
+}
+extension TestProtocol2 {
+  func inExt() -> Assoc { fatalError() }
+}
+struct ConcreteTestProtocol2: TestProtocol2 {
+  func foo() -> some Comparable { 1 }
+  #^OVERRIDE_TestProtocol2^#
+// OVERRIDE_TestProtocol2: Begin completions
+// OVERRIDE_TestProtocol2-NOT: foo()
+// OVERRIDE_TestProtocol2-NOT: inExt()
+// OVERRIDE_TestProtocol2-DAG: Decl[InstanceMethod]/Super:         func bar() -> Assoc {|};
+// OVERRIDE_TestProtocol2-DAG: Decl[InstanceMethod]/Super:         func baz(x: @autoclosure () -> Assoc) -> (Assoc) -> Assoc {|};
+// OVERRIDE_TestProtocol2-DAG: Decl[AssociatedType]/Super:         typealias Assoc = {#(Type)#};
+// OVERRIDE_TestProtocol2-NOT: foo()
+// OVERRIDE_TestProtocol2-NOT: inExt()
+// OVERRIDE_TestProtocol2: End completions
+}
+func testUseTestProtocol2(value: ConcreteTestProtocol2) {
+  value.#^POSTFIX_ConcreteTestProtocol2^#
+// POSTFIX_ConcreteTestProtocol2: Begin completions
+// POSTFIX_ConcreteTestProtocol2-DAG: Keyword[self]/CurrNominal:          self[#ConcreteTestProtocol2#];
+// POSTFIX_ConcreteTestProtocol2-DAG: Decl[InstanceMethod]/CurrNominal:   foo()[#Comparable#];
```

Comment:
I think I understand the issue with override completions, but why does it impact member completion type annotation?

Reply:
Do you mean why this is printed as `Comparable` instead of `some Comparable`? If so, that's because we decided not to print `some` when it was introduced.

Or why `bar()[#ConcreteTestProtocol2.Assoc#]`? This is actually a side effect, but when thinking about `baz({#x: ConcreteTestProtocol2.Assoc#})[#(ConcreteTestProtocol2.Assoc) -> ConcreteTestProtocol2.Assoc#]`,
It's better than `(Comparable) -> Comparable` because using the result doesn't accept arbitrary `Comparable`.

Reply:
> Do you mean why this is printed as Comparable instead of some Comparable? If so, that's because we decided not to print some when it was introduced.

This is what I meant, thanks for explaining. I forgot that, and I misread some of the other test cases to think this was making it inconsistent.